fiddler/datadog_checks/fiddler/check.py

```diff
@@ -37,9 +37,12 @@ def __init__(self, name, init_config, instances):
         self.log.info("with org id : %s", self.org)
         self.log.info("and auth key : %s", self.api_key)
 
-        self.client = fdl.FiddlerApi(url=self.base_url, org_id=self.org, auth_token=self.api_key)
+        self.client = None
 
     def check(self, _):
+        if not self.client:
+            self.client = fdl.FiddlerApi(url=self.base_url, org_id=self.org, auth_token=self.api_key)
```

Comment:
Moved the fiddler client initialization to the check method so that any invalid config values don't block the check from running.

Reply:
Looks good.